src/components/rating/rating.tsx

```diff
@@ -92,6 +105,13 @@ export class Rating
    */
   @Prop({ reflect: true }) required = false;
 
+  /**
+   * Set the maximum number of stars
+   *
+   * @default 5
+   */
+  @Prop() max = 5;
```

Comment:
Do we want to introduce a property here? I think we always want 5 stars for consistency within the design system. Maybe this could be a private variable?